### ISSUE TITLE
web.bale.ai: breakage

### ISSUE THREAD

**Issue by rezad1393** [uBlockOrigin/uAssets#22650]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://web.bale.ai
```

### Description

recently when "uBlock filters – Privacy" is enabled, then https://web.bale.ai website doesnt work.
it shows an empty page.

in logs of addon it shows this:
web.bale.ai##+js(trusted-set-constant, navigator.userAgent, 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36')

clicking on the #@# which I think it adds an exception for that then it works.

### Other extensions used

nothing

### Configuration

uBlock Origin: 1.56.0
Firefox: 123
filterset (summary):
 network: 144634
 cosmetic: 120008
 scriptlet: 40374
 html: 2055
listset (total-discarded, last-updated):
 removed:
  plowe-0: null
 added:
  ublock-annoyances: 6761-11, now
  adguard-generic: 79840-5378, now
  adguard-mobile: 9117-19, now
  adguard-social: 22199-39, now
  fanboy-thirdparty_social: 68-0, now
  IRN-0: 3002-13, 1h.41m
 default:
  user-filters: 2449-0, never
  ublock-filters: 37577-141, now
  ublock-badware: 8006-14, now
  ublock-privacy: 740-0, 5m
  ublock-unbreak: 2280-0, now
  easylist: 81761-322, now
  easyprivacy: 50278-88, now
  urlhaus-1: 9145-0, now
  ublock-quick-fixes: 123-2, now
filterset (user): [array of 2402 redacted]
trustedset:
 added: [array of 11 redacted]
 removed:
  about-scheme
  chrome-scheme
  edge-scheme
  moz-extension-scheme
  opera-scheme
  vivaldi-scheme
  wyciwyg-scheme
switchRuleset:
 added: [array of 3 redacted]
 removed:
  no-large-media: behind-the-scene false
hostRuleset:
 added: [array of 4 redacted]
userSettings:
 advancedUserEnabled: true
 contextMenuEnabled: false
 largeMediaSize: 500
 showIconBadge: false
 webrtcIPAddressHidden: true
hiddenSettings:
 autoCommentFilterTemplate: -
supportStats:
 allReadyAfter: 319 ms (selfie)
 maxAssetCacheWait: 96 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 9
 network:
  bale.ai: 5
  bale.sh: 4

**Comment by rezad1393** [comment]
the fix doesnt work for me.

maybe that !#if !env_mobile is the issue.
I am on desktop.

maybe it can be added as a general exception on that website?

**Comment by rezad1393** [comment]
btw where are my exception are kept?

when I click on that #@# and it gets added to a list?
where I can find those? they are not in my filter or myruels

**Comment by stephenhawk8054** [comment]
`!#if !env_mobile` means `not mobile`

The fix will be auto updated in a few hours, if you update it right now, it hasn't come yet. You can use

```adb
web.bale.ai#@#+js(trusted-set-constant, navigator.userAgent, 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36')
```

in the mean time. Click on uBO icon > &#9881; [Dashboard button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Add the filter(s) in ["My filters" pane](https://github.com/gorhill/uBlock/wiki/Dashboard:-My-filters) > &#10003; Apply changes > Open new tab and test again.

**Comment by rezad1393** [comment]
I forgot to say that I force updated the asset before testing.

**Comment by stephenhawk8054** [comment]
> The fix will be auto updated in a few hours. If you update it right now, it hasn't come yet.

**Comment by rezad1393** [comment]
> btw where are my exception are kept?
> 
> when I click on that #@# and it gets added to a list? where I can find those? they are not in my filter or my rules

when I can see my own exceptions?

**Comment by stephenhawk8054** [comment]
If I remember correctly, those are just temporary, it will go after you close the browser. Add the filter manually as I insttructed above.

**Comment by rezad1393** [comment]
oh it seems to be even more temporary.
it doesnt even need to close the browser.

btw how can I tailor that exception to be more general?
from 
web.bale.ai#@#+js(trusted-set-constant, navigator.userAgent, 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36')

to maybe like this
web.bale.ai#@#+js(trusted-set-constant)

**Comment by stephenhawk8054** [comment]
You can't.

**Comment by gwarser** [comment]
> If I remember correctly, those are just temporary, it will go after you close the browser.

The logger.

**Comment by rezad1393** [comment]
> Fixed, same reason as [#22710 (comment)](https://github.com/uBlockOrigin/uAssets/issues/22710#issuecomment-1993254875).

that ever that fix was, broke web.bale.ai again.
